hkmc2/shared/src/main/scala/hkmc2/MLsCompiler.scala

```diff
@@ -80,13 +82,15 @@ class MLsCompiler(preludeFile: os.Path, mkOutput: ((Str => Unit) => Unit) => Uni
       val elab = Elaborator(etl, wd, newCtx)
       val parsed = mainParse.resultBlk
       val (blk0, _) = elab.importFrom(parsed)
+      val resolver = ImplicitResolver(rtl)
+      resolver.resolveBlk(blk0)(using ImplicitResolver.ICtx.empty)
       val blk = new semantics.Term.Blk(
         semantics.Import(State.runtimeSymbol, runtimeFile.toString) :: blk0.stats,
         blk0.res
       )
       val low = ltl.givenIn:
         new codegen.Lowering()
-          with codegen.LoweringSelSanityChecks
+          // with codegen.LoweringSelSanityChecks
```

Comment:
Is this still necessary?

I briefly tested the runtime of the test. With `codegen.LoweringSelSanityChecks`:
```
[success] Total time: 22 s, completed Apr 19, 2025, 3:18:10 PM
```
Without `codegen.LoweringSelSanityChecks`:
```
[success] Total time: 22 s, completed Apr 19, 2025, 3:20:53 PM
```

Reply:
What do you mean by "still necessary"?

Sanity checks are currently disabled in compiler tests. Of course the running time will be the same: you're running the same thing twice...

Reply:
My plan is to enable light sanity checks by default. Haven't gotten around to it, yet.

Reply:
Why make this change in the first place? We shouldn't be removing the sanity checks mixin.

Reply:
I should mention the context. Previously, some issues in the sanity check led to a stack overflow, and then you introduced this fix. Now it should be reverted, right?

Reply:
Oh, I remember, now! This was so long ago... Yes, if the fix is no longer needed, it should be reverted.